Functions.Templates/Templates/SqlOutputBinding-CSharp-Isolated/.template.config/template.json

```diff
@@ -0,0 +1,71 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Output",
+        "SQL"
+    ],
+    "name": "SqlOutputBindingIsolated",
+    "identity": "Azure.Function.CSharp.Isolated.SqlOutputBinding",
+    "groupIdentity": "Azure.Function.SqlOutputBindingIsolated",
+    "shortName": "sqloutput",
+    "tags": {
+        "language": "C#",
+        "type": "item"
+    },
+    "sourceName": "SqlOutputBindingHttpTriggerCSharp",
+    "symbols": {
+        "namespace": {
+            "description": "namespace for the generated code",
+            "replaces": "Company.Function",
+            "type": "parameter"
+        },
+        "connectionStringSetting": {
+            "description": "The name of the app setting containing your SQL connection string.",
+            "type": "parameter",
+            "defaultValue": "",
+            "replaces": "SqlConnectionString"
```

Comment:
Something to follow up on after this - why is this not the default value? That's what we have in all our samples and is a reasonable name to give so that seems like the better approach.